examples/3l-node/main.rs

```diff
@@ -51,7 +51,7 @@ fn main() {
         local_persistence_path: BASE_DIR.to_string(),
         timezone_config: TzConfig {
             timezone_id: String::from("Europe/Zurich"),
-            timezone_utc_offset_secs: 1 * 60 * 60,
+            timezone_utc_offset_secs: 2 * 60 * 60,
```

Comment:
An idea for a pragmatic solution that we don't have to change the time zone every 6 months, but still a way to test being not on UTC:

Instead of being locate in `Europe/Zurich` we tell the app it's located in `Africa/Tunis`. Tunisia is about on the same latitude as Switzerland, however it does not do daylight savings time: The offset is year-round `1 * 60 * 60` .